"""fixing some errors

Revision ID: f27bf3a474f8
Revises: 0f2257506004
Create Date: 2024-04-06 01:11:42.744284

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = 'f27bf3a474f8'
down_revision = '0f2257506004'
branch_labels = None
depends_on = None


def upgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('fields', schema=None) as batch_op:
        batch_op.add_column(sa.Column('dataformat_id', sa.Integer(), nullable=True))
        batch_op.create_foreign_key(None, 'dataformats', ['dataformat_id'], ['id'])

    # ### end Alembic commands ###


def downgrade():
    # ### commands auto generated by Alembic - please adjust! ###
    with op.batch_alter_table('fields', schema=None) as batch_op:
        batch_op.drop_constraint(None, type_='foreignkey')
        batch_op.drop_column('dataformat_id')

    # ### end Alembic commands ###
